src/app/pages/poll/poll.component.ts

```diff
@@ -100,6 +101,25 @@ export class PollComponent {
     });
   }
 
+  /**
+   * Update the current poll item for this poll on the server
+   *
+   * @param event Click event
+   * @param pollItemId Id of the poll item
+   */
+  onRunPollItem(event: any, pollItemId: number): void {
+    event.stopPropagation();
+    const oldPollItem = this.poll.currentItem;
+    this.poll.currentItem = this.loadingRunPollItemId = pollItemId === this.poll.currentItem ? null : pollItemId;
```

Comment:
Because this reflects the state on the server. Initially (when the poll is freshly created or no poll item was selected yet), the currentItem field of a poll is `null`. So this should also be the state to that we return when an item gets manually stopped by the poll creator.

Reply:
But in this case the condition should read `!this.poll.currentItem`, right?